data_steward/tools/load_vocab.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-
+set -ex
```

Comment:
I like `set -x`, but be sure to end the script with `set +x` or the command line will keep echoing everything even after the script has finished.  

What is your user experience with `set -e`?  I really dislike it since I work from the command line.  When it encounters an error it closes my working terminal and I can't debug.  If you leave it in, I would suggest turning it off at the end of the script also `set +e`.

Reply:
Thanks, I didn't realize these shell options persisted. I will end the script with `set +x`.

I just used `set -e` because I saw it in many of our scripts. I dislike it because it seems to me to abandon ship even when commands just issue a warning. For example, in this script if `cp` warns it's skipping copying a directory or `bq mk` warns of an existing dataset- I think we'd want to proceed. I will remove that option altogether.

Reply:
I didn't know they persisted either until I started tinkering with them in the `run_tests.sh` script.  Woohoo!  We both learned something new!